### ISSUE TITLE
DKIM heading wrong when enabled

### ISSUE THREAD

**Issue by henare** [mlandauer/cuttlefish#204]
The heading says **Improve delivery** and the button is _Disable DKIM_

![selection_001](https://cloud.githubusercontent.com/assets/48945/6548121/a70bbea4-c642-11e4-84b9-7f0fbbe4ddc9.png)

Maybe the heading should be changed if it's enabled because currently it seems to suggest disabling DKIM would improve delivery!

**Comment by bmichelsen** [comment]
Good catch! What if we change the heading to **DKIM signing keys** instead? That way, the heading says something about what the section covers and we can keep the button text as it is.
